homework-3/src/test/java/addDate.java

```diff
@@ -0,0 +1,28 @@
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class addDate {
+    private static final DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd");
+
+    public static void main(String[] args) {
+        Date currentDate = new Date();
+
+        changeDate(currentDate);
+    }
+
+    static void changeDate(Date date) {
+        Calendar neDate = Calendar.getInstance();
+        neDate.setTime(date);
+
+        neDate.add(Calendar.YEAR, 2);
+        neDate.add(Calendar.MONTH, 2);
+        neDate.add(Calendar.DATE, 2);
```

Comment:
that should be in the parameters

so the usage of this method should look like this:
**changeDate(currentDate, daysToAdd, monthsToAdd, yearsToAdd)**